src/test/resources/text/shocked.json

```diff
@@ -1,29 +1,33 @@
 [
-  {
-    "title": "Nani?!"
-  },
-  {
-    "title": "What the?!"
-  },
-  {
-    "title": "Oh No!"
-  },
-  {
-    "title": "Whyyy???"
-  },
-  {
-    "title": "The horror!"
-  },
-  {
-    "title": "OH MY GAH!!"
-  },
-  {
-    "title": "How could this happen?!"
-  },
-  {
-    "title": "Why did you do that??"
-  },
-  {
-    "title": "Hurry quick fix it!"
-  }
-]
\ No newline at end of file
+    {
+        "title": "Nani?!"
+    },
+    {
+        "title": "What the?!"
+    },
+    {
+        "title": "Oh No!"
+    },
+    {
+        "title": "Whyyy???"
+    },
+    {
+        "title": "The horror!"
+    },
+    {
+        "title": "OH MY GAH!!"
+    },
+    {
+        "title": "How could this happen?!"
+    },
+    {
+        "title": "Why did you do that??"
+    },
+    {
+        "title": "Hurry quick fix it!"
+    },
+    {
+        "title": "Dafuq?!",
+        "groupId": "e5feeb14-69e5-4596-8d84-c564c20a9715"
```

Comment:
Would highly recommend this plugin:
https://plugins.jetbrains.com/plugin/9836-randomness
to generate random UUIDs at your leisure